platform-configs/include/novacustom-common.robot

```
@@ -96,6 +96,7 @@ ${DTS_FIRMWARE_FLASHING_SUPPORT}=                   ${TRUE}
 ${DTS_EC_FLASHING_SUPPORT}=                         ${TRUE}
 ${ACPI_DRIVER_SUPPORT}=                             ${TRUE}
 ${ACPI_CAMERA_SWITCH_SUPPORT}=                      ${FALSE}
+${INTEL_CBNT_SUPPORT}=                              ${TRUE}
```

IIRC you also have to add it to `platform-configs/include/default.robot`, or it will FAIL on other platforms instead of a nice skip

---

good point, added to https://github.com/Dasharo/open-source-firmware-validation/pull/856/commits/8de23b5e1fcf9dccc4a82c214043744278342b6c